test/cli/harness/peering.go

```diff
@@ -0,0 +1,37 @@
+package harness
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/ipfs/kubo/config"
+)
+
+type Peering struct {
+	From int
+	To   int
+}
+
+func NewRandPort() int {
+	n := rand.Int()
+	return 3000 + (n % 1000)
+}
```

Comment:
Can you make `NewRandPort` private ? I see your point about not wanting to change too much in `peering_test` but I still don't like this code, I don't want other tests to start using this in the future. :)